active-sessions/src/main.rs

```diff
@@ -20,25 +18,27 @@ pub struct ActiveSessionsService;
 impl ActiveSessions for ActiveSessionsService {
     async fn add_user(
         &self,
-        request: Request<UserData>,             //Userdata from auth-service
-    )-> Result<Response<String>, Status> {      
+        request: Request<UserData>,
+    ) -> Result<Response<UserDataResponse>, Status> { 
 
+        println!("Received request to add user");
         let user_data = request.into_inner();
-
         let session_token = generate_session_token();
         
         let redis_url = "redis://redis:6379/";
```

Comment:
Why do we have here a connection?
```suggestion
       I would have connection to database in main, as it will be stil open which can make this program run faster than connect and reconect with every new request
```